READMEen.md

```diff
@@ -0,0 +1,98 @@
+
+---
+
+# 🎥 RanczoKlipy Bot 🤖
+<img src="Avatar.png" alt="Avatar" width="315"/>
+
+**RanczoKlipy Bot** is a highly customizable Telegram bot designed to manage and process video clips from the popular TV series "Ranczo." 🎬 The bot allows users to search for specific quotes, manage their own video clips, and perform various administrative tasks related to user management and content moderation.
```

Comment:
nie chce mi sie czytac tego wszystkiego ale jak chcesz miec tip top to se mozesz przez to przepuscic https://www.deepl.com/en/write

Reply:
To jest z GPT4o to jeszcze lepiej a to są detale